install/updateVersion.sh

```diff
@@ -25,16 +25,36 @@ cd ${ROOT}
 # just run the old version
 ${ROOT}/install/updateVersion_orig.sh "$@"
 
+echo "$@"
+
+DEST_DIR=${ROOT}
+while getopts :n:d:a: arg; do
```

Comment:
what does n argument do?

Reply:
Removal of this flag broke the script :-( 
`":d:a"`  does not parse a which was critical...